ui/pages/onboarding-flow/create-password/create-password.js

```diff
@@ -58,18 +62,77 @@ export default function CreatePassword({
   }, [password, confirmPassword, passwordError, confirmPasswordError]);
 
   const handlePasswordChange = (passwordInput) => {
-    let error = '';
     let confirmError = '';
-    if (passwordInput && passwordInput.length < 8) {
-      error = t('passwordNotLongEnough');
+    let passwordStrengthDescription = '';
+    let passwordStrengthInput;
+    const passwordEvaluation = zxcvbn(passwordInput);
+
+    switch (passwordEvaluation.score) {
+      case 0:
+        passwordStrengthInput = t('passwordStrength', [
+          <span
+            key={passwordEvaluation.score}
+            className="create-password__form__weak"
+          >
+            {t('weak')}
+          </span>,
+        ]);
+        passwordStrengthDescription = t('passwordStrengthDescription');
+        break;
+      case 1:
+        passwordStrengthInput = t('passwordStrength', [
+          <span
+            key={passwordEvaluation.score}
+            className="create-password__form__weak"
+          >
+            {t('weak')}
+          </span>,
+        ]);
+        passwordStrengthDescription = t('passwordStrengthDescription');
+        break;
+      case 2:
+        passwordStrengthInput = t('passwordStrength', [
+          <span
+            key={passwordEvaluation.score}
+            className="create-password__form__weak"
+          >
+            {t('weak')}
+          </span>,
+        ]);
+        passwordStrengthDescription = t('passwordStrengthDescription');
+        break;
+      case 3:
+        passwordStrengthInput = t('passwordStrength', [
+          <span
+            key={passwordEvaluation.score}
+            className="create-password__form__average"
+          >
+            {t('average')}
+          </span>,
+        ]);
+        passwordStrengthDescription = t('passwordStrengthDescription');
```

Comment:
[nit] since this is the same for all cases except `strong` maybe we should initialize it to `t('passwordStrengthDescription');` and set it to `''` in case 4?

Reply:
@adonesky1 Fixed in: 31b1950541693058091725c294105bf99a39b4fa